src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2778,6 +2791,11 @@ internal static ConstantValue GetIsOperatorConstantResult(TypeSymbol operandType
 
             Debug.Assert((object)operandType != null);
 
+            operandCouldBeNull =
+                operandCouldBeNull &&
+                operandType.CanContainNull() && // a value type is never null
```

Comment:
Yes. I meant the comment to give the reader a hint why the method is called, not to precisely specify the called method.